src/theme/breakpoints.scss

```diff
@@ -0,0 +1,14 @@
+$breakpoints: (
+  xxsmall: 280px,
```

Comment:
I think it's helpful devs in the future to know why these are breakpoints, e.g. if they're associated with a particular device screen width or common screen resolution.

I'd recommend adding a comment for as many as possible, but not spending too much time on this (i.e. less than 30 mins)